src/System.Diagnostics.DiagnosticSource/src/ApiCompatBaseline.txt

```diff
@@ -0,0 +1,2 @@
+MembersMustExist : Member 'System.Diagnostics.DiagnosticListener.DefaultListener.get()' does not exist in the implementation but it does exist in the contract.
```

Comment:
Pull request  #6817 has been merged, so you can remove the baselines for DiagnosticSource and DiagnosticListener.